### PR TITLE
SRVOCF-346: Add a section on fine-tuning log levels in Node.js functions

### PR DESCRIPTION
**Pull request by rh-max**
For versions 4.6+
Jira: https://issues.redhat.com/browse/SRVOCF-346
Docs preview: https://deploy-preview-35588--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-reference-guide.html#serverless-nodejs-context-object-reference-log_serverless-functions-reference-guide

### CODE REVIEW

**Comment by rh-max** [comment]
Hey @lance could you please review?
This one is somewhat lob-sided in comparison to https://github.com/openshift/openshift-docs/pull/35587 , because here I've only documented the `log` property. After this goes out, I can document the rest of properties for Node.js templates in the same manner as for TypeScript templates, should I open a Jira for that?

**Comment by rh-max** [comment]
> Hey @lance could you please review?
> This one is somewhat lob-sided in comparison to #35587 , because here I've only documented the `log` property. After this goes out, I can document the rest of properties for Node.js templates in the same manner as for TypeScript templates, should I open a Jira for that?

Never mind this comment, I've just realized that we do have them documented for Node.js, just in a different place. Hold on with that review please, I'll restructure the thing.

**Comment by rh-max** [comment]
@lance PTAL

**Review by lance — APPROVED** [review]
/lgtm

**Review by mikemckiernan — COMMENTED** [review]
One style guide comment and a content question.  If there's anything I can clarify, please let me know.

File: modules/serverless-nodejs-context-object-reference.adoc [hunk 1]
```diff
@@ -34,6 +34,8 @@ $ curl http://example.com
 {"level":30,"time":1604511655265,"pid":3430203,"hostname":"localhost.localdomain","reqId":1,"msg":"Processing customer"}
 ----
 
+You can change the log level to one of `fatal`, `error`, `warn`, `info`, `debug`, `trace` or `silent`. To do that, change the value of `logLevel` by assigning one of these values to the environment variable `FUNC_LOG_LEVEL` using the `config` command.
```
[the remark this answers is not in the document]
Comment (rh-max) [reply modules/serverless-nodejs-context-object-reference.adoc:37]: @mikemckiernan "I kind of expected the first sentence to follow the first code block. I'm assuming that customers can use..."  - You are right, and they can use any other of the specified log levels in addition to `debug` and `fatal`. So it's not at odds with the code block.
"or the second sentence, there is no other mention of logLevel on the page" - Good catch, I've changed the text per the suggestion from Jefferson from QE. Now, the reader can see how to change the enigmatic `logLevel`.

**Pull request merged**